src/utils.ts

```diff
@@ -29,7 +29,7 @@ export function resolveNopeRefsFromKeys(options: string[], context?: Record<stri
 }
 
 export function every(arr: any[], predicate: (value: any) => boolean) {
-  return arr.filter((value) => !predicate(value)).length === 0;
+  return arr.every(predicate);
```

Comment:
Thank you for the PR! I dont think we really need the wrapper here anymore - we can just use .every on the original array.  Whats your thought on that? 😄

Reply:
haha, doesn't make sense at all, sorry about that ;)